starlight/templates/base.html

```diff
@@ -1,45 +1,32 @@
 <!DOCTYPE html>
 {% load i18n %}
 {% load render_bundle webpack_static from webpack_loader %}
+{% load static %}
 <html>
-
 <head>
     <meta charset="UTF-8">
-    <title>
-        {% block title %}{% trans "Starlight" %}{% endblock %}
-    </title>
 
     {% render_bundle 'main' 'css' %}
 
-    {% block extrahead %}{% endblock %}
+    <title>Starlight</title>
+
+    <div class="ui inverted menu">
+
+        <a class="{% ifequal viewname 'home' %}active blue item{% else %}black item{% endifequal %}" href="{% url 'home' %}"> Home </a>
+        <a class="black item">Individual Pages</a>
+        <a class="{% ifequal viewname 'login' %}active blue item{% else %}black item{% endifequal %}" href="{% url 'login' %}"> Login </a>
```

Comment:
You could even do a check here that sees if the user is logged in, then only displays the "Logout" option if that is the case. Otherwise, it displays "Login". :)